scribble-lib/scribble/acmart.rkt

```diff
@@ -9,7 +10,31 @@
          (for-syntax racket/base))
 
 (provide/contract
- [title (->* (pre-content?) (#:short pre-content?) content?)]
+ [title (->* (pre-content?)
+             (#:short pre-content?
+              #:tag (or/c string? (listof string?) #f)
+              #:tag-prefix (or/c string? module-path? #f)
+              #:style (or/c style? string? symbol? #f)
+              #:version (or/c string? #f)
+              #:date (or/c string? #f))
+             title-decl?)]
+ [author (->* () () #:rest (listof pre-content?)
+              paragraph?)]
+ [authorinfo (->* (string?)
+                  ((or/c string? multiarg-element? (listof string?) (listof multiarg-element?))
+                   (or/c string? (listof string?))
```

Comment:
I'm down for that. If that's the case then we really can just drop `authorinfo` altogether and just use `author`, which I'm okay with.